data-visualization/FBPlots.jl

```diff
@@ -1,3 +1,5 @@
+include!("../data-utils/Utils.jl")
```

Comment:
Why? As far as I know, this is unnecessary. Unless you had some future design ideas in mind

Reply:
Moved geometry constants into there